schema.sql

```diff
@@ -44,4 +44,6 @@ CREATE TABLE visits (
   vets_id INT references vets(id),
   animals_id INT references animals(id),
   date_of_visits DATE
-);
\ No newline at end of file
+);
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
```

Comment:
- [ ] Kindly add indexes that are used to improve performance inside the `schema.sql` file as any change in the schema should be saved in this file as per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=Any%20changes%20you%20made%20to%20your%20database%20schema%20should%20be%20added%20to%20the%20schema.sql%20file%20and%20commited%20in%20a%20new%20pull%20request.).